react-backend/app.js

```diff
@@ -39,7 +39,8 @@ app.use((err, req, res) => {
   res.send({ error, message });
 });
 
-const port = process.env.PORT || 5000;
+const port = process.env.NODE_ENV === 'test' ? 5010 : process.env.PORT || 5000;
```

Comment:
What is this for? So you can have the app running while you're running tests?

Reply:
Yup

Reply:
I stole this for my pr 😈